src/hwloc/hwloc_base_util.c

```diff
@@ -659,6 +659,24 @@ unsigned int prte_hwloc_base_get_nbobjs_by_type(hwloc_topology_t topo, hwloc_obj
     }
 
 #if HWLOC_API_VERSION >= 0x20000
+    /* if the type is NUMA, then we need to only count the
+     * CPU NUMAs and ignore the GPU NUMAs as we only deal
+     * with CPUs at this time */
+    if (HWLOC_OBJ_NUMANODE == target) {
+        unsigned width, w;
+        hwloc_obj_t obj;
+
+        width = hwloc_get_nbobjs_by_type(topo, target);
+        rc = 0;
+        for (w=0; w < width; w++) {
+            obj = hwloc_get_obj_by_type(topo, HWLOC_OBJ_NUMANODE, w);
+            if (200 > obj->os_index) {
```

Comment:
Yeah - I'll also make it a `#define` named constant so it will be easier to change should someone put lots and lots of GPUs (or other non-CPU domains) on a system.